eqllib/analytics/collection/T1005-txt-files-with-command-line-redirection.toml

```
@@ -0,0 +1,22 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies Koadic's use of redirecting command output to plaintext files."
+id = "7c7f3114-7bdd-4477-a4e0-b5105b6babd8"
+name = "Creation of TXT File via cmd.exe with Command Line Redirection"
```

The _via_ and _with_ are tricky to combine. Maybe something like this?
```suggestion
name = "WMI Execution with Command Line Redirection"
```